src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4849,4 +4849,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ElseClauseRequiredWithWhenClause" xml:space="preserve">
     <value>An 'else' clause is required when a 'when' clause is present.</value>
   </data>
-</root>
+  <data name="ERR_SemiOrLBraceOrArrowExpected" xml:space="preserve">
+    <value>{ or ; or =&gt; expected</value>
```

Comment:
Even for methods the errors are only
```C#
int M1() =
// CS1002: ; expected
// CS1519: Invalid token `=` in class, struct, or interface member declarartion
```
Shouldn't we keep the the more verbose error here?